phoenix/slackbot/models.py

```diff
@@ -38,19 +36,14 @@ def save(self, force_insert=False, force_update=False, using=None,
     def dedicated_channel_name(self):
         date = self.outage.created.date().strftime('%y%m%d')
         systems = ','.join([system.name for system in self.outage.systems_affected.all()])[:9]
-        #offset = Outage.objects.filter(  # Ignore PEP8Bear
-            #created__day=self.outage.created.date().day,  # Ignore PyCommentedCodeBear
-            #pk__lt=self.outage.pk,
-            #systems_affected__in=self.outage.systems_affected.all(),
-        #).count()
+        offset = Outage.objects.filter(  # Ignore PEP8Bear
+            created__day=self.outage.created.date().day,  # Ignore PyCommentedCodeBear
+            pk__lt=self.outage.pk,
+            systems_affected__in=self.outage.systems_affected.all(),
```

Comment:
how does this work if you're checking a list of systems in another list of systems? won't it fail if you have a day with an infrastructure + search outage, and then an infrastructure + booking outage? cause both will use `infrastru` in the channel name but won't match with this query

Reply:
Yes it will but the channel name is combination of all systems as well, so it would be different channel name.

Reply:
it looks like it's just the first 9 characters, so whether it's `infrastructure-search` or `infrastructure-booking`, the part in the channel will be `infrastru`, no?

Reply:
Yes that's true. Solutions are either to change the channel name format or remove the option to specify multiple systems (it's never been used anyway because we can't implement that in slack). As this multiple systems affected feature has never been used it doesn't bring any value and unnecessary complicates things, that would be the most reasonable solution. What do you think?

Reply:
Yup, let's remove it

Reply:
Devided into two commits. 
[First commit](https://github.com/kiwicom/phoenix/pull/9/commits/fce98ea6785c5651806c8bccb773f2404d312877) will migrate the m2m field into o2m. [Second commit](https://github.com/kiwicom/phoenix/pull/9/commits/0f71af31d063ba7e84496a592b9068e87ef82813) will remove the old m2m field completely.